server/events/event_parser_test.go

```diff
@@ -1215,6 +1215,7 @@ func TestParseAzureDevopsRepo(t *testing.T) {
 		VCSHost: models.VCSHost{
 			Hostname: "dev.azure.com",
 			Type:     models.AzureDevops,
+			VcsToken: "azuredevops-token",
```

Comment:
This is probably a naive question, is this vcs token similar to the headers listed here?

https://github.com/runatlantis/atlantis/blob/33bc28fa855e4734391f3dbe6e7d524968ee4da4/server/controllers/events/events_controller.go#L38-L42

Reply:
This is a different Header. The header above needs to be on the incoming webhook request. This PR resolves the need for an additional authentication header on the call to git.